src/bandersnatch/defaults.conf

```diff
@@ -9,6 +9,7 @@ master = https://pypi.org
 proxy =
 download-mirror =
 download-mirror-no-fallback = false
+; allow-non-https = false
```

Comment:
If we're explicitly setting to false, it can be a default. But can leave it commented too ...

"Explicit is better than implicit" is how I always think about these things.